base/string.jl

```diff
@@ -2,7 +2,11 @@
 
 print(io::IO, x) = show(io, x)
 print(io::IO, xs...) = for x in xs print(io, x) end
```

Comment:
should print lock also/instead? It seems strange that println(xs...) would lock but print(xs...) would not.